graphql/resolvers/exerciseCommentCrud.ts

```diff
@@ -44,3 +45,28 @@ export const addExerciseComment = async (
     data: { authorId, content, exerciseId, parentId, userPic }
   })
 }
+
+export const editExerciseComment = async (
+  _parent: void,
+  { id, content }: MutationEditExerciseCommentArgs,
+  context: Context
+) => {
+  const authorId = context.req.user?.id
+  if (!authorId) throw new Error('User should be logged in')
+
+  const exerciseComment = await prisma.exerciseComment.findUnique({
+    where: {
+      id
+    }
+  })
+
+  if (exerciseComment?.authorId !== authorId)
+    throw new Error('Comment is not by user')
+
+  return prisma.exerciseComment.update({
```

Comment:
It would be nice to include the date too.

Reply:
Actually, I though prisma updates that automatically in the  "updatedAt" field. If that's the case we wont need to manually update the time right?

Reply:
You're right.